### PR TITLE
feat: add Chai framework for unit testing

### LINKED ISSUE #37: Problem: tests framework is not installed

### PR DESCRIPTION
**Pull request by accompany1205**
resolves #37

### CODE REVIEW

File: tests/StreamManager.test.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+const { ethers } = require("hardhat");
+
+describe("StreamManager", function () {
+  before(async () => {
+    const [admin, payer, payee1, payee2] = await ethers.getSigners()
+    this.admin = admin
+    this.payer = payer
+    this.payee1 = payee1
+    this.payee2 = payee2
+    this.token = '0xA02f6adc7926efeBBd59Fd43A84f4E0c0c91e832' // USDT on mumbai testnet
```
Comment (markholdex) [inline tests/StreamManager.test.js:11]: @accompany1205 not sure how we are going to get test tokens for ourselves too.

Comment (accompany1205) [reply tests/StreamManager.test.js:11]: It's just a setup for future use. We can use our own if we have.

**Comment by zolotokrylin** [comment]
@CTAPCKPIM please rename this PR to match the following guidelines https://www.conventionalcommits.org/en/v1.0.0/

File: tests/StreamManager.test.js (hunk 1 above)
Comment (zolotokrylin) [reply tests/StreamManager.test.js:11]: @markholdex this is out of the scope. This is just to set up the testing framework

**Review by zolotokrylin — CHANGES_REQUESTED** [review]
@accompany1205 just rename the PR and make sure the commit has similar name. Thanks

**Comment by accompany1205** [comment]
Yeah, I will

**Comment by accompany1205** [comment]
Is it okay?

**Comment by accompany1205** [comment]
Please help me if you don't like it

**Comment by zolotokrylin** [comment]
@accompany1205 did you read the link I have shared with you above? 

```
Commits MUST be prefixed with a type, which consists of a noun, feat, fix, etc., followed by the OPTIONAL scope, OPTIONAL !, and REQUIRED terminal colon and space.
The type feat MUST be used when a commit adds a new feature to your application or library.
The type fix MUST be used when a commit represents a bug fix for your application.
```

**Comment by accompany1205** [comment]
Yeah, I will try with it.

**Pull request merged**